test/e2e/performanceprofile/functests/Z_deconfig/deconfig.go

```diff
@@ -0,0 +1,27 @@
+package Z_deconfig
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	testutils "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils"
+	testclient "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/client"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/namespaces"
+	nodeInspector "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/node_inspector"
+)
+
+// This test suite is designed to perform cleanup actions that should occur after all test suites have been executed.
+
+var _ = Describe("Deconfig", func() {
+	It("Should delete the node inspector and its namespace", func() {
+		err := nodeInspector.Delete(testclient.Client, testutils.NodeInspectorNamespace, testutils.NodeInspectorName)
```

Comment:
The deletion of the namespace two line below, would remove the pod inspector automatically

Reply:
This is what I did previously. I counted on the namespace to delete the node inspector and its related object.
I found out that the ClusterRole was not deleted and lived past the cleanup, since it is cluster scoped.
I could instead call a delete function solely for the ClusterRole, but I don't see any big difference here.

Reply:
Deleting more objects, which is what `node_inspector.Delete` does, is inherently more fragile though, because we need to handle more error paths robustly. However we need `node_inspector.Delete` anyway, so we may very much call it here.